src/rates.ts

```diff
@@ -208,9 +209,15 @@ export const getExchangeRates = async (
         delete doc.updated
         const cleanDoc = asCouchDoc(asObject(asString))(doc)
         for (const pair of Object.keys(cleanDoc.doc)) {
+          // Only save USD rates that do not inlcude zeroRates and
+          // falbackConstantRates to redis and couchdb.
           if (
-            currencyCodeArray(pair).includes('iso:USD') ||
-            preferredCryptoFiatPairs.includes(pair)
+            (currencyCodeArray(pair).includes('iso:USD') ||
+              preferredCryptoFiatPairs.includes(pair)) &&
+            !(
+              pair in currencyCodeMaps.zeroRates ||
+              pair in currencyCodeMaps.fallbackConstantRates
```

Comment:
This one will work for the pairs explicitly listed in the fallbackConstantRates map but it will not filter out other pairs that are implicitly served by these hardcoded values